standards_lab/api/views.py

```diff
@@ -140,15 +160,33 @@ def delete(self, *args, **kwargs):
 
 class ProjectDownloadFile(View):
     def get(self, request, *args, **kwargs):
+        type = kwargs["type"]
```

Comment:
Sorry missed this the first time round, we're overloading a python keyword here, I think it is generally better practice to avoid doing this in case someone later on tries/needs  to do `type(moo)`  something like `project_input_type` ? (trying to avoid "file type" as that is already a loaded term).